proxy/pkg/config/config.go

```diff
@@ -19,11 +20,12 @@ type Config struct {
 
 	// Global bucket
 
-	PrimaryCluster          string `default:"ORIGIN" split_words:"true" yaml:"primary_cluster"`
-	ReadMode                string `default:"PRIMARY_ONLY" split_words:"true" yaml:"read_mode"`
-	ReplaceCqlFunctions     bool   `default:"false" split_words:"true" yaml:"replace_cql_functions"`
-	AsyncHandshakeTimeoutMs int    `default:"4000" split_words:"true" yaml:"async_handshake_timeout_ms"`
-	LogLevel                string `default:"INFO" split_words:"true" yaml:"log_level"`
+	PrimaryCluster                string `default:"ORIGIN" split_words:"true" yaml:"primary_cluster"`
+	ReadMode                      string `default:"PRIMARY_ONLY" split_words:"true" yaml:"read_mode"`
+	ReplaceCqlFunctions           bool   `default:"false" split_words:"true" yaml:"replace_cql_functions"`
+	AsyncHandshakeTimeoutMs       int    `default:"4000" split_words:"true" yaml:"async_handshake_timeout_ms"`
+	LogLevel                      string `default:"INFO" split_words:"true" yaml:"log_level"`
+	ControlConnMaxProtocolVersion string `default:"3" split_words:"true" yaml:"control_conn_max_protocol_version"` // Numeric Cassandra OSS protocol version or DseV1 / DseV2
```

Comment:
We should put this as the highest version possible for the default because we will kind of infer the protocol version on the non control connections from the control connection versions.